configure.ac

```diff
@@ -466,12 +473,28 @@ AC_ARG_ENABLE(network,
               AS_HELP_STRING([--disable-network],
                              [Disable networking features (modem, ipx)]),,
               enable_network=yes)
-AC_CHECK_HEADER(SDL_net.h,have_sdl_net_h=yes,)
+AC_ARG_ENABLE(sdl_net_static,
+              AC_HELP_STRING([--enable-sdl-net-static],
+                             [Link SDL2_Net statically (not recommended)]),
+              enable_sdl_net_static=yes,
+              enable_sdl_net_static=no)
+AC_CHECK_HEADER(SDL_net.h, have_sdl_net_h=yes,)
 AC_CHECK_LIB(SDL2_net, SDLNet_Init, have_sdl_net_lib=yes, , )
 AC_MSG_CHECKING([whether networking features will be enabled])
 if test x$enable_network = xyes ; then
   if test x$have_sdl_net_lib = xyes -a x$have_sdl_net_h = xyes ; then
-    LIBS="$LIBS -lSDL2_net"
+    if test x$enable_sdl_net_static = xyes ; then
+      case "$host" in
+        *-*-darwin*)
+          LIBS="$LIBS /usr/local/lib/libSDL2_net.a"
```

Comment:
*Just a side note (not blocking this review).*

We haven't actually tested if `-Wl,-Bstatic -l<libname> <libflags> -Wl,-Bdynamic` works on macOS with brew packages! When --enable-sdl-static was being implemented, I didn't knew about `-Wl,-Bstatic -Wl,-Bdynamic` option). Perhaps we can get rid of having separate case for Darwin.